client/src/components/App.jsx

```diff
@@ -27,10 +23,10 @@ productDetail-skeleton
   return (
     <div id="App">
       <Product product={product} setProduct={setProduct} />
-      <StarTemplate />
+      {/* <StarTemplate />
       <QAModule product={product} />
       <RelatedProductsList product={product} />
-      <RatingsAndReviews product={product} />
+      <RatingsAndReviews product={product} /> */}
     </div>
   );
```

Comment:
Please uncomment the components inside App.jsx.

Reply:
Done!